src/main/java/org/magicdgs/readtools/cmd/argumentcollections/TrimmerPluginArgumentCollection.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Daniel Gomez-Sanchez
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package org.magicdgs.readtools.cmd.argumentcollections;
+
+import org.magicdgs.readtools.cmd.RTStandardArguments;
+
+import org.broadinstitute.barclay.argparser.Argument;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Argument collection to use with {@link org.magicdgs.readtools.cmd.plugin.TrimmerPluginDescriptor}.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public abstract class TrimmerPluginArgumentCollection {
```

Comment:
Extracting this generates warnings in `DefaultDocWorkUnitHandler`:

```
16:04:02.924 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for userTrimmerNames, presumably because the field is inaccessible
16:04:02.924 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for userTrimmerNames, presumably because the field is inaccessible
16:04:02.925 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for userTrimmerNames, presumably because the field is inaccessible
16:04:02.953 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for disabledTrimmers, presumably because the field is inaccessible
16:04:02.953 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for disabledTrimmers, presumably because the field is inaccessible
16:04:02.954 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for disabledTrimmers, presumably because the field is inaccessible
16:04:02.979 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for disableAllDefaultTrimmers, presumably because the field is inaccessible
16:04:02.980 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for disableAllDefaultTrimmers, presumably because the field is inaccessible
16:04:02.980 WARN  DefaultDocWorkUnitHandler - Could not access the field definition for backtrace while searching for disableAllDefaultTrimmers, presumably because the field is inaccessible
```

Reply:
Open #350 for tracking this.